localstorage-sync.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Deriv</title>
+        <meta charset="utf-8" />
+        <meta name="referrer" content="origin" />
+        <script>
+            window.addEventListener('message', function (message) {
+                var allowed_origins = [
+                    'https://deriv.app',
+                    'https://staging.deriv.app',
+                    'https://app.deriv.com',
+                    'https://app.deriv.me',
+                    'https://app.deriv.be',
+                    'https://staging-app.deriv.com',
+                    'https://staging-app.deriv.me',
+                    'https://staging-app.deriv.be',
+                    'https://test-app.deriv.com',
+                ];
+
+                if (allowed_origins.includes(message.origin)) {
+                    switch (message.data.key) {
+                        case 'active_loginid': {
+                            if (localStorage.getItem(message.data.key) !== message.data.value) {
+                                localStorage.setItem(message.data.key, message.data.value);
```

Comment:
isn't it doing the same thing for both the cases

Reply:
I just used the same thing as smarttrader and binary bot. I guess it can have other keys in the message.data which we don't need that's why we are just searching for these two keys in the object.